src/main/java/seedu/letsgethired/ui/SelectView.java

```diff
@@ -0,0 +1,28 @@
+package seedu.letsgethired.ui;
+
+import static java.util.Objects.requireNonNull;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.TextArea;
+import javafx.scene.layout.Region;
+
+/**
+ * A ui for the select view that displays the notes of a selected application.
+ */
+public class SelectView extends UiPart<Region> {
+    private static final String FXML = "SelectView.fxml";
+    @FXML
+    private TextArea selectView;
+
+    /**
+     * Creates a {@code SelectView}.
+     */
+    public SelectView() {
+        super(FXML);
+    }
+
+    public void displayNotes(String notes) {
```

Comment:
Perhaps we can add a JavaDoc header comment here to pass the test?

Reply:
Its failing because of tests failing. The test fails because I have yet to integrate Notes Command into my code. Aprup's PR has to be cleared for me to continue.

![image](https://github.com/AY2324S1-CS2103T-W17-2/tp/assets/93871694/7eca1cbc-c4bc-4af0-9bef-8fc1a67a229e)

Reply:
Just merged Aprup's PR! Thanks You Wei for the update!